cfme/services/myservice/ssui.py

```diff
@@ -230,14 +230,9 @@ def launch_vm_console(self, catalog_item):
 @MiqImplementationContext.external_for(MyService.retire, ViaSSUI)
 def retire(self):
     view = navigate_to(self, 'Retire')
-    view.retire.click()
-    view = self.create_view(MyServicesView)
-    assert wait_for(
-        lambda: view.is_displayed, delay=3, num_sec=300,
-        message="waiting for view to be displayed"
-    )
-    # TODO - remove sleep when BZ 1518954 is fixed
-    time.sleep(10)
+    if self.appliance.version < "5.10":
```

Comment:
@izapolsk in 5.10 confirmation pop-up window for retiring the service is not appearing so service is retiring without confirmation.
In 5.9 confirmation pop-up window appear after clicking on retire.
I added check.
```view.retire.click()``` is part of confirmation popup and for 5.10 its not needed as i said confirmation pop-up window is missing.

Reply:
**Required**

@niyazRedhat you've had to describe this to both @sshveta and @izapolsk in this PR - this information needs to be in-line here as a comment block, listing the BZ's involved.  Otherwise other contributors aren't aware of what's happening here, or if/when this version check should be removed.